pkg/data/components/SymbolicatorProcessor.yaml

```diff
@@ -0,0 +1,60 @@
+kind: SymbolicatorProcessor
+name: Symbolicator Processor
+style: processor
+type: base
+status: development
+version: v0.1.0
+summary: A processor that will symbolicate JavaScript stack traces using source maps.
+description: |
+  This processor is used to symbolicate JavaScript stack traces using source maps.
+tags:
+  - category:processor
+  - service:collector
+  - signal:OtelTraces
+ports:
+  - name: TracesIn
+    direction: input
+    type: OTelTraces
+  - name: TracesOut
+    direction: output
+    type: OTelTraces
+properties:
+  - name: SourceMapStore
+    type: string
+    subtype: oneof(file_store, s3_store, gcs_store)
+    validations:
+      - oneof(file_store, s3_store, gcs_store)
```

Comment:
What is the default value?

Reply:
Hmm, didn't notice it before but looks like [file_store](https://github.com/honeycombio/opentelemetry-collector-symbolicator/blob/main/symbolicatorprocessor/factory.go#L34) is the default value. I'll add that.

Reply:
I don't think we should allow file_store as an option since thst requires a rather complex setup at the HTP-pipeline level to enable a build of their code to push the files into a place that the container can mount.

Better to stick with S3/gcp and force them to configure it that way.